langchain_math.py

```diff
@@ -1,17 +1,18 @@
 # Initialise imports
-from langchain.agents import create_openai_functions_agent, AgentExecutor
 from langchain import hub
+from langchain.agents import AgentExecutor, create_openai_functions_agent
 from langchain_openai import ChatOpenAI
```

Comment:
This seems like an example/sample code. We should move this to `examples/` instead of at the root folder